libs/blocks/marketo/marketo.js

```diff
@@ -199,5 +200,6 @@ export default function init(el) {
     callback: (target) => {
       loadMarketo(target, formData);
     },
+    options: { rootMargin: `${ROOT_MARGIN}px` },
```

Comment:
If root margin is not used anywhere else could store it as just a string and omit the template literal?

Reply:
You could, however there's 5 other blocks using this format and it makes sense to keep `ROOT_MARGIN` as an int.